mobile/src/app.tsx

```diff
@@ -51,12 +51,13 @@ export class App extends Component<AppProps> {
     }
 
     private onBackPress = () => {
-        const { dispatch, nav } = this.props;
-        if (nav.index === 0) {
+        const { dispatch, nav: currentState } = this.props;
+        const popAction = NavigationActions.pop({ n: 1 });
```

Comment:
won't it crash if index === 0?

Reply:
or it's handled with getStateForAction and condition? A question when, if we check for index AND do pop instead of back, will it work? Don't really like getStateForAction call but I can live with it :)

Reply:
@santee 
First of all, index here is a tab index (0 == feeds tab). :)
When there is nothing to pop, navigation reducer will just return the same state.

Reply:
got it